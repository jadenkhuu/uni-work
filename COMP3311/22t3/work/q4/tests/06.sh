psql racing -c "select * from q4('oo') order by horse;"
