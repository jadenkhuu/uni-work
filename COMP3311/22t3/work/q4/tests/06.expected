       horse       | average 
-------------------+---------
 Book Law          |   14909
 David Hasselhoofs |  129470
 Hoof Hearted      |   13333
 Hoofy Heart       |     200
 Hooves            |   27157
 Whykickamoocow    |   65210
(6 rows)
